ansible/roles/test/files/ptftests/arista.py

```diff
@@ -387,6 +398,31 @@ def parse_bgp_route(self, output, expects):
 
         return set(expects) == prefixes
 
+    def parse_supported_show_lacp_command(self, lacp_help):
+        """
+        'show lacp neighbor' is deprecated by 'show lacp peer' in high EOS versions,
+        so if 'show lacp neighbor' is supported, use 'show lacp neighbor'
+        otherwise use 'show lacp peer' instead
+        Args:
+            lacp_help (str): result of command 'show lacp ?', be like :
+                                   aggregates  Display info about LACP aggregates
+                                   counters    Display LACP counters
+                                   internal    Display information internal to the Link Aggregation Control Protocol
+                                   neighbor    Display identity and info about LACP neighbor(s)
+                                   peer        Display identity and info about LACP neighbor(s)
+                                   sys-id      Display System Identifier used by LACP
+                                   $           list end
+                                   <1-2000>    Channel Group ID(s)
+        Returns:
+            str: rest command of 'show lacp ', neighbor or peer
+        """
+
+
```

Comment:
Can you remove the extra blank line here?

Reply:
refined